autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/AlertType.java

```diff
@@ -1,5 +1,16 @@
 package com.sequenceiq.periscope.api.model;
 
 public enum AlertType {
-    METRIC, TIME, PROMETHEUS, LOAD
+    METRIC, TIME, PROMETHEUS, LOAD;
+
+    @Override
+    public String toString() {
+        switch (this) {
```

Comment:
what is your opinion about that you convert only the different value and you are using the `.name()` any other case?

Reply:
ok, updated to use .name()